api/src/controllers/auth.ts

```diff
@@ -213,6 +220,22 @@ router.post(
 	respond
 );
 
+router.get(
+	'/',
+	asyncHandler(async (req, res, next) => {
+		const providers = toArray(env.AUTH_PROVIDERS);
+		const providerData = providers
+			.filter((provider) => provider && env[`AUTH_${provider.toUpperCase()}_DRIVER`])
+			.map((provider) => ({
+				name: provider,
+				driver: env[`AUTH_${provider.toUpperCase()}_DRIVER`],
+			}));
+		res.locals.payload = { data: env.AUTH_PROVIDERS ? providerData : null };
```

Comment:
We should include whether or not the default auth is enabled

Reply:
Not sure I understand? We need default enabled to continue to support the existing oauth flow. After we move to the new oauth, however, we can disable default

Reply:
This must've made sense to me at some point..